app/indexers/hyrax/valkyrie_work_indexer.rb

```diff
@@ -26,5 +29,11 @@ def to_solr # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Met
     def suppressed?(resource)
       Hyrax::ResourceStatus.new(resource: resource).inactive?
     end
+
+    def admin_set_label(resource)
+      return "" if resource.admin_set_id.blank?
```

Comment:
If it is blank should this return nil instead?  If it is nil then the field won't get stored in solr, but I think it will if it is empty string.

Reply:
DONE